dtde/clusterlib.py

```diff
@@ -126,7 +127,7 @@ def generate_cluster_start_task(
             name="SPARK_JUPYTER_PORT", value=spark_jupyter_port),
         batch_models.EnvironmentSetting(
             name="SPARK_WEB_UI_PORT", value=spark_web_ui_port),
-    ]
+    ] + get_docker_credentials()
```

Comment:
? Add this on a new line?

Reply:
you'd have to have a \ at the end of the line which would look ugly here